zebra/zebra_rnh.c

```diff
@@ -780,11 +780,10 @@ void zebra_evaluate_rnh(struct zebra_vrf *zvrf, afi_t afi, int force,
 	if (p) {
 		/* Evaluating a specific entry, make sure it exists. */
 		nrn = route_node_lookup(rnh_table, p);
-		if (nrn && nrn->info)
```

Comment:
you've changed the logic here: we only want to evaluate this node if it's got context associated with it

Reply:
Yes I agree with Mark here, this change is incorrect. The route table does maintain some extra nodes which does not have the context. This will lead to crashes.

Reply:
I think the message is "the existing code is clear, and there's no value in changing it" ?